nodejs/src/mcp-client.js

```
@@ -0,0 +1,768 @@
+  }
+
+  async getVariables(sessionId) {
+    try {
+      const result = await this.sendCustomToolRequest('get_local_variables', { session_id: sessionId });
+
+      // Handle different response formats
+      let variables;
+      if (typeof result === 'string') {
+        variables = JSON.parse(result);
+      } else if (result.content && result.content[0] && result.content[0].text) {
+        variables = JSON.parse(result.content[0].text);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)